tests/5.0/loop/test_loop_lastprivate.c

```diff
@@ -1,4 +1,4 @@
-//===--- test_loop_lastprivate.c ----------------------------------------------===//
```

Comment:
I believe this line may have been deleted accidentally? Otherwise looks good.

Reply:
Thanks for pointing out - now re-added.